### PR TITLE
PRL-2497 Terms and Conditions

### PR DESCRIPTION
**Pull request by JamesCutts00**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRL-2497

### Change description ###

Updated Terms and Conditions

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/main/steps/terms-and-conditions/content.test.ts [hunk 1]
```diff
@@ -7,163 +7,110 @@ jest.mock('../../app/form/validation');
 
 const en = {
   title: 'Terms and conditions',
-  thisPage:
-    'By using this service you’re agreeing to these terms of use. This includes the <a class="govuk-link" href="/privacy-policy">privacy policy</a>.',
-  termsAndConditionsUsers: 'Terms and conditions for professional users',
-  termsAndConditionsProfessional:
-    "If you are a professional user, you'll be asked to agree to the additional terms and conditions of the service after you create a MyHMCTS account.",
+  byAccessingThisService:
+    'By accessing this service (‘the Service’), you’re agreeing to the following terms of use. This includes the <a class="govuk-link" href="/privacy-policy">privacy policy</a>.',
```
Comment (AlokDatta) [inline src/main/steps/terms-and-conditions/content.test.ts:11]: ‘the Service’ - it should be service name, i.e. Private law. You can refer to cookie banner page to know about how we can put service name.

File: src/main/steps/terms-and-conditions/content.test.ts [hunk 2]
```diff
@@ -7,163 +7,110 @@ jest.mock('../../app/form/validation');
...
-  goodwill: 'goodwill or reputation',
-  tangibleProperty: 'tangible property',
-  intangibleProperty: 'intangible property, including loss, corruption or damage to data or any computer system',
-  wastedTime: 'wasted management or office time',
-  liable: 'We may still be liable for:',
-  death: 'death or personal injury arising from our negligence',
-  fraudulentMisrepresentation: 'fraudulent misrepresentation',
-  anyOtherLiability: 'any other liability which cannot be excluded or limited under applicable law',
-  contactUsFurtherInfo: '<a class="govuk-link" href="/contact-us">Contact us</a> for further information.',
+  thisServiceIs: 'This Service is managed by Her Majesty’s Courts and Tribunals service.',
```
Comment (AlokDatta) [inline src/main/steps/terms-and-conditions/content.test.ts:18]: Her Majesty's? Dont think that's appropriate now. Put this as HM for now

File: src/main/steps/terms-and-conditions/content.test.ts [hunk 3]
```diff
@@ -7,163 +7,110 @@ jest.mock('../../app/form/validation');
...
-    'intangible property, including loss, corruption or damage to data or any computer system (in Welsh)',
-  wastedTime: 'wasted management or office time (in Welsh)',
-  liable: 'We may still be liable for: (in Welsh)',
-  death: 'death or personal injury arising from our negligence (in Welsh)',
-  fraudulentMisrepresentation: 'fraudulent misrepresentation (in Welsh)',
-  anyOtherLiability: 'any other liability which cannot be excluded or limited under applicable law (in Welsh)',
-  contactUsFurtherInfo: '<a class="govuk-link" href="/contact-us">Contact us</a> for further information. (in Welsh)',
+  title: 'Terms and conditions {in welsh}',
+  byAccessingThisService:
+    'By accessing this service (‘the Service’), you’re agreeing to the following terms of use. This includes the <a class="govuk-link" href="/privacy-policy">privacy policy</a>. {in welsh}',
```
Comment (AlokDatta) [inline src/main/steps/terms-and-conditions/content.test.ts:65]: same as above.

File: src/main/steps/terms-and-conditions/content.test.ts [hunk 4]
```diff
@@ -7,163 +7,110 @@ jest.mock('../../app/form/validation');
...
+  theUseOfTheService:
+    'The use of the Service includes the Manage Organisation and Manage Cases applications (sometimes known as ‘MyHMCTS’) and Judicial Case Manager (JCM). {in welsh}',
+  theTermYouIncludes:
+    'The term ‘you’ includes yourself and all users of the Service in your organisation. The terms ‘we’ and ‘us’ refer to Her Majesty’s Courts and Tribunals Service. {in welsh}',
+  youAlsoAgreeThat: 'You also agree that you have a legitimate need to use the Service. {in welsh}',
+  whoWeAre: 'Who we are {in welsh}',
+  thisServiceIs: 'This Service is managed by Her Majesty’s Courts and Tribunals service. {in welsh}',
```
Comment (AlokDatta) [inline src/main/steps/terms-and-conditions/content.test.ts:72]: Same as above

File: src/main/steps/terms-and-conditions/content.ts [hunk 5]
```diff
@@ -2,163 +2,110 @@ import { TranslationFn } from '../../app/controller/GetController';
 
 const en = {
   title: 'Terms and conditions',
-  thisPage:
-    'By using this service you’re agreeing to these terms of use. This includes the <a class="govuk-link" href="/privacy-policy">privacy policy</a>.',
-  termsAndConditionsUsers: 'Terms and conditions for professional users',
-  termsAndConditionsProfessional:
-    "If you are a professional user, you'll be asked to agree to the additional terms and conditions of the service after you create a MyHMCTS account.",
+  byAccessingThisService:
+    'By accessing this service (‘the Service’), you’re agreeing to the following terms of use. This includes the <a class="govuk-link" href="/privacy-policy">privacy policy</a>.',
```
Comment (AlokDatta) [inline src/main/steps/terms-and-conditions/content.ts:6]: same as test file

File: src/main/steps/terms-and-conditions/content.ts [hunk 6]
```diff
@@ -2,163 +2,110 @@ import { TranslationFn } from '../../app/controller/GetController';
...
+  theUseOfTheService:
+    'The use of the Service includes the Manage Organisation and Manage Cases applications (sometimes known as ‘MyHMCTS’) and Judicial Case Manager (JCM).',
+  theTermYouIncludes:
+    'The term ‘you’ includes yourself and all users of the Service in your organisation. The terms ‘we’ and ‘us’ refer to Her Majesty’s Courts and Tribunals Service.',
```
Comment (AlokDatta) [inline src/main/steps/terms-and-conditions/content.ts:10]: same as test file

File: src/main/steps/terms-and-conditions/content.ts [hunk 7]
```diff
@@ -2,163 +2,110 @@ import { TranslationFn } from '../../app/controller/GetController';
...
-    'intangible property, including loss, corruption or damage to data or any computer system (in Welsh)',
-  wastedTime: 'wasted management or office time (in Welsh)',
-  liable: 'We may still be liable for: (in Welsh)',
-  death: 'death or personal injury arising from our negligence (in Welsh)',
-  fraudulentMisrepresentation: 'fraudulent misrepresentation (in Welsh)',
-  anyOtherLiability: 'any other liability which cannot be excluded or limited under applicable law (in Welsh)',
-  contactUsFurtherInfo: '<a class="govuk-link" href="/contact-us">Contact us</a> for further information. (in Welsh)',
+  title: 'Terms and conditions {in welsh}',
+  byAccessingThisService:
+    'By accessing this service (‘the Service’), you’re agreeing to the following terms of use. This includes the <a class="govuk-link" href="/privacy-policy">privacy policy</a>. {in welsh}',
```
Comment (AlokDatta) [inline src/main/steps/terms-and-conditions/content.ts:60]: same as test file

File: src/main/steps/terms-and-conditions/content.ts [hunk 8]
```diff
@@ -2,163 +2,110 @@ import { TranslationFn } from '../../app/controller/GetController';
...
+  theUseOfTheService:
+    'The use of the Service includes the Manage Organisation and Manage Cases applications (sometimes known as ‘MyHMCTS’) and Judicial Case Manager (JCM). {in welsh}',
+  theTermYouIncludes:
+    'The term ‘you’ includes yourself and all users of the Service in your organisation. The terms ‘we’ and ‘us’ refer to Her Majesty’s Courts and Tribunals Service. {in welsh}',
```
Comment (AlokDatta) [inline src/main/steps/terms-and-conditions/content.ts:64]: same as test file

**Review by AlokDatta — CHANGES_REQUESTED** [review]
Hello James, couple of minor comments you need to update in both main and test file.

File: src/main/steps/terms-and-conditions/content.test.ts (hunk 1 above)
Comment (JamesCutts00) [reply src/main/steps/terms-and-conditions/content.test.ts]: changed to private law service

File: src/main/steps/terms-and-conditions/content.test.ts (hunk 2 above)
Comment (JamesCutts00) [reply src/main/steps/terms-and-conditions/content.test.ts]: changed to hm

File: src/main/steps/terms-and-conditions/content.test.ts (hunk 3 above)
Comment (JamesCutts00) [reply src/main/steps/terms-and-conditions/content.test.ts]: changed to private law service

File: src/main/steps/terms-and-conditions/content.ts (hunk 5 above)
Comment (JamesCutts00) [reply src/main/steps/terms-and-conditions/content.ts]: changed to private law service

File: src/main/steps/terms-and-conditions/content.ts (hunk 6 above)
Comment (JamesCutts00) [reply src/main/steps/terms-and-conditions/content.ts]: changed to hm

File: src/main/steps/terms-and-conditions/content.ts (hunk 7 above)
Comment (JamesCutts00) [reply src/main/steps/terms-and-conditions/content.ts]: changed to private law service

File: src/main/steps/terms-and-conditions/content.ts (hunk 8 above)
Comment (JamesCutts00) [reply src/main/steps/terms-and-conditions/content.ts]: changed to hm

**Pull request merged**